### DISCUSSION TITLE
7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#200] [General]
# 7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3

C/1680 V1, also known as Kirch's Comet, Newton's Comet or Great Comet of 1680 is a comet that in 1680 passed only 930,000 km away from the Sun, so it managed to obtain a very long tail and visible even during the day. The comet was discovered by Gottfried Kirch and its course observed by Eusebio Kino. Isaac Newton used the comet to test and verify Kepler's laws.

https://celestia.mobi/addon?item=7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3

**Comment by RGDPS** [comment]
How
